src/custom/Feedback/FeedbackButton.tsx

```diff
@@ -0,0 +1,250 @@
+        <FeedbackMessage isOpen={isOpen}>
+          <SuccessIcon width={'32'} height={'32'} />
+          We got your concern. Thank you!
+        </FeedbackMessage>
+      ) : (
+        <>
+          <FeedbackButton onClick={handleFeedback}>Feedback</FeedbackButton>
+
+          <ModalCard
+            onClose={() => {}}
+            open={true}
+            closeComponent={
+              <CloseButton onClick={() => setIsOpen(false)}>
+                <CloseIcon fill="#F6F8F8" />
+              </CloseButton>
+            }
+            actions={
+              <div
+                style={{
+                  display: 'flex',
+                  alignItems: 'center'
+                }}
+              >
+                <div style={{ display: 'flex', alignItems: 'center' }}>
+                  <FormControlLabel
+                    label={''}
+                    style={{
+                      color: 'white',
+                      fontSize: '2px',
+                      marginLeft: '0px',
+                      marginRight: '5px'
+                    }}
+                    control={
+                      <Checkbox
+                        style={{ color: '#F6F8F8' }}
+                        checked={isChecked}
+                        onChange={handleCheckboxChange}
+                      />
+                    }
+                  />
+                  <Typography style={{ fontSize: '0.8rem', marginRight: '10px', color: 'white' }}>
+                    We may email you for more information or updates
+                  </Typography>
+                </div>
+                <FeedbackSubmitButton
+                  type="submit"
+                  disabled={!(messageValue && isChecked)}
+                  style={{ backgroundColor: !(messageValue && isChecked) ? '#b0bec5' : '#00B39F' }}
+                  onClick={handleSubmit}
+                >
+                  Send
+                </FeedbackSubmitButton>
+              </div>
+            }
+            leftHeaderIcon={<FeedbackIcon width={'24'} height={'24'} fill="#F6F8F8" />}
+            title="Feedback"
+            helpArea={
+              <Tooltip placement="top" title={tooltipContent} arrow>
+                <div style={{ marginRight: '0.5rem' }}>
+                  <QuestionIcon fill="#F6F8F8" />
+                </div>
+              </Tooltip>
+            }
+            helpText={'Help'}
+            content={
+              <FeedbackForm>
+                <FeedbackOptions>
+                  {feedbackData?.map((item) => (
+                    <FeedbackOptionButton
+                      key={item.label}
+                      type="button"
+                      onClick={() => {
+                        setCategory(item);
+                      }}
+                      style={{
+                        borderBottom: category?.label === item.label ? '2px solid #00B39F' : '',
+                        background: category?.label === item.label ? '#fff' : '',
```

Comment:
themeeeeee

Reply:
yes yes i wrote that in description that dark mode is in progress